internal/capability/auditor.go

```diff
@@ -96,9 +96,21 @@ func (capAuditor *CapAuditor) RunAudits(ctx context.Context) error {
 			// run function/method by name
 			// NOTE: The signature for this method MUST be:
 			// func Fn(context.Context) error
-			m := reflect.ValueOf(&audit).MethodByName(function)
-			in := []reflect.Value{reflect.ValueOf(ctx)}
-			m.Call(in)
+			auditFn := audits.New(ctx, function,
+				audits.WithClient(audit.client),
+				audits.WithNamespace(audit.namespace),
+				audits.WithOperatorGroupData(&audit.operatorGroupData),
+				audits.WithSubscription(&audit.subscription),
+				audits.WithTimeout(int(audit.csvWaitTime)),
```

Comment:
I really like this! My only comment is that this single file would go better with the others and the package being called capability. Then ideally for each feature composing capabilities we would have a clear named file like operator_install.go, operand_install.go, seamless_update.go, backup.go, restore.go, data_insights.go etc. Just to make things obvious to read.

Reply:
I can go with that.